src/features/courses/course-controller.ts

```diff
@@ -3362,6 +3363,63 @@ class CourseController {
             return result;
         });
     }
+
+    // Sends an email to the professor associated with the course.
+    async emailProfessor(options: EmailProfOptions): Promise<boolean> {
+        const coursePromise = Course.findOne({
+            where: {
+                id: options.courseId,
+            },
+            include: [{
+                model: User,
+                as: 'instructor',
+                attributes: ['id', 'firstName', 'lastName', 'email', 'preferredEmail'],
+                where: {
+                    active: true
+                },
+                required: true,
+            }],
+        });
+
+        const gradePromise = StudentGrade.findOne({
+            where: {
+                userId: options.student.id,
+                courseWWTopicQuestionId: options.question.id,
+            },
+            attributes: ['randomSeed']
+        });
+
+        const [course, grade] = await Promise.all([coursePromise, gradePromise]);
+
+        if (_.isNil(course) || _.isNil(course.instructor)) {
+            throw new WrappedError('Could not find an instructor for this course.');
+        }
+
+        if (_.isNil(grade)) {
+            throw new WrappedError('Could not find a grade associated with the problem this student is trying to ask about.');
+        }
+
+        const poorMansTemplate = `
+Hello Professor ${course.instructor.lastName},
+
+    Your student ${options.student.firstName} ${options.student.lastName} is asking for help with
+Problem ${options.question.problemNumber+1} (ID#${options.question.id}) in the Topic ${options.topic.name} (ID#${options.topic.id})
+The WeBWorK path for this problem is ${options.question.webworkQuestionPath} and the seed value is ${grade.randomSeed}.
```

Comment:
We should not put the webwork path or seed in here, too much info and if we add a reply to student from this email (which I think you did) we will be exposing bad things!

Furthermore it is like putting a jira ticket in a message, no one knows what it means until they look at it anyway therefore it's not even particularly useful

Reply:
I guess I'll remove this and we'll just have to remember to update it with @doombot-exe 's stuff after the fact.